### PR TITLE
Adding Planner, ToDoItem and updated Project filetype icons. Updating FabricCDN url to latest datecode.

### PR DESCRIPTION
**Pull request by bigbadcapers**
* [x] Adding `planner`, `todoitem` and updated Project **filetype icons**. Updating FabricCDN url to latest datecode.
* [x] Code is up-to-date with the `master` branch
* [x] Tested in Storybook
* [x] You've run `yarn change` locally
* [x] Ran yarn lage to update published CDN URL string

### CODE REVIEW

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/bf0099d4ecd4c241c90832b2430f3c1a5b76d93a'>bf0099d4ecd4c241c90832b2430f3c1a5b76d93a</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/BaseButton-mount_base.html">639</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/BaseButton-mount.html">637</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Breadcrumb-mount_base.html">1662</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Breadcrumb-mount.html">1675</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Checkbox-mount_base.html">1667</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Checkbox-mount.html">1655</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/CheckboxBase-mount_base.html">1457</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/CheckboxBase-mount.html">1483</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ChoiceGroup-mount_base.html">2944</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ChoiceGroup-mount.html">2977</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ComboBox-mount_base.html">661</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ComboBox-mount.html">644</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/CommandBar-mount_base.html">6150</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/CommandBar-mount.html">6159</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ContextualMenu-mount_base.html">12577</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ContextualMenu-mount.html">12792</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/DefaultButton-mount_base.html">719</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/DefaultButton-mount.html">770</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/DetailsRow-mount_base.html">2229</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/DetailsRow-mount.html">2188</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/DetailsRowFast-mount_base.html">2140</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/DetailsRowFast-mount.html">2179</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/DetailsRowNoStyles-mount_base.html">2011</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/DetailsRowNoStyles-mount.html">2019</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Dialog-mount_base.html">2779</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Dialog-mount.html">2766</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/DocumentCardTitle-mount_base.html">226</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/DocumentCardTitle-mount.html">231</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Dropdown-mount_base.html">1973</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Dropdown-mount.html">1938</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/FocusTrapZone-mount_base.html">1128</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/FocusTrapZone-mount.html">1149</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/FocusZone-mount_base.html">1042</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/FocusZone-mount.html">1057</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedList-mount_base.html">41591</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedList-mount.html">41381</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedList-virtual-rerender_base.html">19947</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedList-virtual-rerender.html">19984</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">50575</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedList-virtual-rerender-with-unmount.html">50418</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedListV2-mount_base.html">225</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedListV2-mount.html">225</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedListV2-virtual-rerender_base.html">215</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedListV2-virtual-rerender.html">215</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">224</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">222</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/IconButton-mount_base.html">1095</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/IconButton-mount.html">1070</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Label-mount_base.html">331</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Label-mount.html">342</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Layer-mount_base.html">2719</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Layer-mount.html">2735</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Link-mount_base.html">384</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Link-mount.html">396</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/MenuButton-mount_base.html">929</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/MenuButton-mount.html">957</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/MessageBar-mount_base.html">21278</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/MessageBar-mount.html">21203</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Nav-mount_base.html">1947</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Nav-mount.html">1995</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/OverflowSet-mount_base.html">758</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/OverflowSet-mount.html">780</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Panel-mount_base.html">1762</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Panel-mount.html">1819</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Persona-mount_base.html">741</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Persona-mount.html">726</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Pivot-mount_base.html">859</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Pivot-mount.html">854</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/PrimaryButton-mount_base.html">838</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/PrimaryButton-mount.html">837</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Rating-mount_base.html">4547</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Rating-mount.html">4640</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/SearchBox-mount_base.html">914</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/SearchBox-mount.html">917</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Shimmer-mount_base.html">1908</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Shimmer-mount.html">1873</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Slider-mount_base.html">1316</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Slider-mount.html">1303</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/SpinButton-mount_base.html">2849</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/SpinButton-mount.html">2840</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Spinner-mount_base.html">392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Spinner-mount.html">379</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/SplitButton-mount_base.html">1803</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/SplitButton-mount.html">1825</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Stack-mount_base.html">399</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Stack-mount.html">416</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/StackWithIntrinsicChildren-mount_base.html">851</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/StackWithIntrinsicChildren-mount.html">844</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/StackWithTextChildren-mount_base.html">2602</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/StackWithTextChildren-mount.html">2585</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/SwatchColorPicker-mount_base.html">6013</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/SwatchColorPicker-mount.html">6193</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/TagPicker-mount_base.html">1459</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/TagPicker-mount.html">1440</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Text-mount_base.html">394</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Text-mount.html">378</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/TextField-mount_base.html">951</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/TextField-mount.html">928</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ThemeProvider-mount_base.html">831</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ThemeProvider-mount.html">838</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ThemeProvider-virtual-rerender_base.html">587</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ThemeProvider-virtual-rerender.html">584</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1280</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1247</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Toggle-mount_base.html">631</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/Toggle-mount.html">605</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/buttonNative-mount_base.html">197</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30079/perf-test/buttonNative-mount.html">197</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-file-type-icons-c16199c3-d573-42c6-85af-8e38a51c0741.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-file-type-icons-c16199c3-d573-42c6-85af-8e38a51c0741.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30079&commitId=33512ec70bdb0db809448a143e909985e47901be&env=prod&clientType=FLUENTUI)


  <details><summary><b>PeoplePicker</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Normal with text.suggestion-  a .chromium.png|129| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Comment (bigbadcapers) [reply change/@fluentui-react-file-type-icons-c16199c3-d573-42c6-85af-8e38a51c0741.json:1]: The diff shows a missing people avatar in a control that's unrelated to the code changed in this PR. Marked as "noise"

**Review by tomi-msft — APPROVED** [review]

**Comment by bigbadcapers** [comment]
Thank you for taking a look at this PR, @tomi-msft ! Could I get your help with merging? It appears protected from letting me do so directly.

**Pull request merged**